hera_mc/cm_handling.py

```diff
@@ -388,10 +365,8 @@ def show_other_connections(self, connection_dossier, already_shown):
         already_shown: list of shown connections from show_connections
         """
 
-        for k, v in connection_dossier.iteritems():
-            if k in self.non_class_connection_dossier_entries:
-                continue
-            elif k in already_shown:
+        for k, v in connection_dossier['connections'].iteritems():
+            if k in already_shown:
                 continue
             else:
```

Comment:
The `else` isn't really needed since the non-`else` branch does a `continue`. Simplest is

```
for k, v in connection_dossier['connections'].iteritems():
   if k not in already_shown:
       print(v, v.start_date, v.stop_date)
```